ci-automation/test_env.sh

```diff
@@ -1,11 +1,23 @@
 #!/usr/bin/env bash
 
-# Values: start|end
+on_exit() {
+    echo "$?"
+		docker rm -f centos-test-env
+    exit
+}
+
 command=$1
 
-if [[ "$command" = "start" ]]; then
-	docker build -t lioramilbaum/ubuntu .
-	docker run -it --privileged=true -v $(pwd)/../:/root/ember-csi lioramilbaum/ubuntu
-elif [[ "$command" = "end" ]]; then
-	docker rm $(docker ps -aq)
+if [[ "$command" = "up" ]]; then
+	docker build -t centos-test-env .
+	docker run -d --privileged -d -e 'container=docker' \
+		-v /sys/fs/cgroup:/sys/fs/cgroup:rw -v $(pwd)/../:/root/ember-csi \
+		--name centos-test-env centos-test-env
+	docker exec -i centos-test-env /root/ember-csi/ci-automation/container_up.sh
+	sudo rm -rf .vagrant/machines
```

Comment:
If you need to make sure that this is cleaned, move it to on_exit()
Otherwise, make this line can be dropped?

Reply:
@ifireball It was your advice in the first place, but, maybe I didn't understand you. The entrypoint of the base image is running init. I do not want to override it. RIght?